src/main/java/org/opentripplanner/middleware/controllers/api/OtpUserController.java

```diff
@@ -1,103 +1,19 @@
-
-        // Add the user token route BEFORE the regular CRUD methods
-        // (otherwise, /fromtoken requests would be considered
-        // by spark as 'GET user with id "fromtoken"', which we don't want).
-        ApiEndpoint modifiedEndpoint = baseEndpoint
-            // Get user from token.
-            .get(path(ROOT_ROUTE + TOKEN_PATH)
-                .withDescription("Retrieves an OtpUser entity using an Auth0 access token passed in an Authorization header.")
-                .withResponseType(persistence.clazz),
-                this::getUserFromRequest, JsonUtils::toJson
-            )
-
-            // Options response for CORS for the token path
-            .options(path(TOKEN_PATH), (req, res) -> "");
-
-        // Add the regular CRUD methods after defining the /fromtoken route.
-        super.buildEndpoint(modifiedEndpoint);
-    }
-
-    /**
-     * Before creating/storing a user in MongoDB, create the user in Auth0 and update the {@link OtpUser#auth0UserId}
-     * with the value from Auth0.
-     */
-    @Override
-    OtpUser preCreateHook(OtpUser user, Request req) {
-        com.auth0.json.mgmt.users.User auth0UserProfile = createNewAuth0User(user, req, this.persistence);
-        return updateAuthFieldsForUser(user, auth0UserProfile);
-    }
-
-    @Override
-    OtpUser preUpdateHook(OtpUser user, OtpUser preExistingUser, Request req) {
-        validateExistingUser(user, preExistingUser, req, this.persistence);
-        return user;
-    }
-
-    /**
-     * Before deleting the user in MongoDB, attempt to delete the user in Auth0.
-     */
-    @Override
-    boolean preDeleteHook(OtpUser user, Request req) {
-        try {
-            deleteAuth0User(user.auth0UserId);
-        } catch (Auth0Exception e) {
-            logMessageAndHalt(req, 500, "Error deleting user.", e);
-            return false;
-        }
-        return true;
-    }
-
-    /**
-     * HTTP endpoint to get the {@link OtpUser} entity, if it exists, from an {@link Auth0UserProfile} attribute
-     * available from a {@link Request} (this is the case for '/api/secure/' endpoints).
-     */
-    private OtpUser getUserFromRequest(Request req, Response res) {
-        Auth0UserProfile profile = Auth0Connection.getUserFromRequest(req);
-        OtpUser user = profile.otpUser;
-
-        // If the OtpUser object is null, it is most likely because it was not created yet,
-        // for instance, for users who just created an Auth0 login and have an Auth0UserProfile
-        // but have not completed the account setup form yet.
-        // For those users, the OtpUser profile would be 404 not found (as opposed to 403 forbidden).
-        if (user == null) {
-            logMessageAndHalt(req, HttpStatus.NOT_FOUND_404, String.format(NO_USER_WITH_AUTH0_ID_MESSAGE, profile.auth0UserId), null);
-        }
-        return user;
+    protected OtpUser getUserProfile(Auth0UserProfile profile) {
+        return profile.otpUser;
```

Comment:
When I run this branch locally with https://github.com/opentripplanner/otp-react-redux/pull/174/files I get an error of being unable to create a user because of a missing required field `isEmailVerified`.

Reply:
https://github.com/opentripplanner/otp-react-redux/pull/174 states that is depends on https://github.com/opentripplanner/otp-react-redux/pull/200 for the `isEmailVerified` field.

Reply:
How does opentripplanner/otp-react-redux#200 supply the `isEmailVerified` field? Isn't it just concerned with `planParamsToQueryAsync`? 

Nevertheless, I think this gets to the point that PR descriptions must contain info on how to run/test the code therein.

Reply:
My bad, I meant to say https://github.com/opentripplanner/otp-react-redux/pull/199 (just missed by one!) I updated https://github.com/opentripplanner/otp-react-redux/pull/174's description to mention that too.